Assets/Scripts/Guts/CadaverGutsManager.cs

```diff
@@ -34,22 +52,30 @@ void Update()
         
     }
 
+    //Useful to get last spline for backtrack (playerSpawner.cs)
+    public Spline GetLastCreatedSpline()
+    {
+        int lastIndex = _splineContainer.Splines.Count - 1;
+        return _splineContainer.Splines[lastIndex];
```

Comment:
This exists

https://learn.microsoft.com/en-us/dotnet/api/system.linq.enumerable.last?view=net-8.0

```suggestion
        return _splineContainer.Splines.Last();
```